apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -176,14 +180,32 @@ export class AppMessagesConverter {
 
 		const attachments = this._convertAppAttachments(message.attachments);
 
+		let _id = message.id;
```

Comment:
What about:
```
const _id = !isPartial && !message.id ? Random.id() : message.id
```

wonder if it's a bit better considering we're sacrificing a bit of read vs a bunch of {}

Reply:
I've considered it, but my reasoning was to avoid repeated tests, as I'd need to do that same thing with the other variables (`!isPartial && ...`)